recipes/pyspoa/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+git clone --recursive https://github.com/nanoporetech/pyspoa.git
```

Comment:
Can you maybe create a release tarball with the submodule included? This way we can create a backup of the tarball.

Reply:
Done.